src/desktop/config.coffee

```diff
@@ -13,6 +13,7 @@ module.exports =
   APPLICATION_NAME: 'force-staging'
   APPLY_URL: 'http://apply.artsy.net'
   APP_URL: 'http://localhost:3004'
+  ARTIST_INSIGHTS_V2_ENABLED: false
```

Comment:
idk if we need this, we'd want to deploy the AB test and most likely would not want to turn it off. The new component looks great!